apps/meteor/server/startup/migrations/v273.ts

```diff
@@ -0,0 +1,13 @@
+import { MongoInternals } from 'meteor/mongo';
+
+import { addMigration } from '../../lib/migrations';
+
+// Remove Deprecated Omnichannel Queue Collection
+addMigration({
+	version: 273,
+	async up() {
+		// Remove collection
+		const { mongo } = MongoInternals.defaultRemoteCollectionDriver();
+		await mongo.db.dropCollection('rocketchat_omnichannel_queue');
```

Comment:
if for some reason the collection doesn't exists, this will fail and halt the start up

Reply:
What would be the right way of removing it then? 👀 
(This collection has been for quite a while, someone would have to update from a version older than 1.5 years for it to not be there [The col was there when i joined haha])

But it could happen if someone manually rolls back the migration lock number 🤔

Reply:
maybe just wrap it into a try/catch :awesome-face:

Reply:
![image](https://user-images.githubusercontent.com/11577696/176699600-c5d3f8cc-d066-468b-b13c-9258d91a14a8.png)